DemoApp/DemoApp/Views/ModalPage.xaml.cs

```diff
@@ -1,61 +0,0 @@
-using DemoApp.Constant;
```

Comment:
Why did you delete this file? Is this not used anymore?

Reply:
Duplicated files